app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
+  before_action :configure_permitted_parameters, if: :devise_controller?
+  
   protect_from_forgery with: :exception
+
+  protected
+
+  def configure_permitted_parameters
+  	temp = [:first_name, :last_name, :email, :password, :password_confirmation]
```

Comment:
According to http://stackoverflow.com/questions/2122380/using-do-block-vs-brackets
because there is a small operation such as pushing field to an array, I prefer brackets. But I'm not sure about it.